resources/referers.yml

```diff
@@ -275,6 +275,7 @@ social:
       - instagram.com
       - l.instagram.com
       - com.instagram.android # Instagram for Android app
+      - lnk.bio # Instagram bio link tool that allows Instagram users to share multiple links through a single landing page
```

Comment:
Right on the home page, it says "Supercharge your link in bio on Instagram, **_TikTok, YouTube_** ..."